cobalt/dom/html_media_element.h

```diff
@@ -229,7 +229,10 @@ class HTMLMediaElement : public HTMLElement,
 
   void ConfigureMediaControls();
 
+  void ReportCurrentTimeToMediaSource();
```

Comment:
Should we consider to add a comment for what this new function does?

Reply:
Done.